example.yaml

```diff
@@ -3,12 +3,13 @@
 
 # The addresses on which to listen for statsd metrics. These are
 # formatted as URLs, with schemes corresponding to valid "network"
-# arguments on https://golang.org/pkg/net/#Listen. Currently, only udp
-# and tcp (including IPv4 and 6-only) schemes are supported.
-# This option supersedes the "udp_address" and "tcp_address" options.
+# arguments on https://golang.org/pkg/net/#Listen.Currently, only udp
```

Comment:
There's a missing comma after `udp` at the end of the line here, and there should be a space after the period after `Listen`